webapps/frontend/app/viewmodels/services/statistics.js

```diff
@@ -0,0 +1,47 @@
+// Copyright (C) 2017 iNuron NV
+//
+// This file is part of Open vStorage Open Source Edition (OSE),
+// as available from
+//
+//      http://www.openvstorage.org and
+//      http://www.openvstorage.com.
+//
+// This file is free software; you can redistribute it and/or modify it
+// under the terms of the GNU Affero General Public License v3 (GNU AGPLv3)
+// as published by the Free Software Foundation, in version 3 as it comes
+// in the LICENSE.txt file of the Open vStorage OSE distribution.
+//
+// Open vStorage is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY of any kind.
+/*global define */
+/**
+ * Service to help with backend related tasks
+ */
+define([
+    'jquery', 'knockout',
+    'ovs/api', 'ovs/shared'
+], function ($, ko, api, shared) {
+    function StatisticsService() {
+        var self = this;
+
+
+        // Functions
+        /**
+         * Fetch the currently saved graphite credentials
+         * @returns {Deferred}
+         */
+        self.graphiteStatistics = function() {
+            return api.get('ovs/statistics/graphite');
```

Comment:
This path is not right. Looks like a configuration path instead of an API one?
Edit: revisiting this file because all saving is done using the postSupportSettings so this service is not used at all?